external/graphdb-service.js

```diff
@@ -127,16 +127,7 @@ class GraphdbService {
                             ?s ?p ?o
                           }
                         }`;
-        let nquads = await this.construct(query);
-
-        if (nquads.length) {
```

Comment:
Where is this moved?

Reply:
I moved it to DataService resolve function. I think Nquads canonisation should be independent of the implementation used.

https://github.com/OriginTrail/ot-node/blob/a1655330e81be529de19ca7d47f6a4dd78e6ec84/modules/service/data-service.js#L140